### ISSUE TITLE
Scroll To Top light switch controlling Page Progress Position visibility

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#762]
**Description**
The **Scroll To Top** light switch setting is changing the visibility of the **Page Progress Position** field.

**Steps to reproduce**
1. From CP edit form page, select the **Appearance** tab.
2. Enable the **Display Page Progress** light switch. **Page Progress Position** field displays as expected.
3. Disable the **Scroll To Top** light switch. **Page Progress Position** field is hidden.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.9
- Craft version: Pro 3.7.33
- Multi-site: No

**Comment by engram-design** [comment]
Whoops! Clearly a copy-paste error. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.10](https://github.com/verbb/formie/releases/tag/1.5.10)
